From 0dc14149ba74a6b552a5bc7d23ed9a541dd4267c Mon Sep 17 00:00:00 2001
From: Luciano Coelho <coelho@ti.com>
Date: Thu, 16 Feb 2012 00:55:05 +0200
Subject: [PATCH 01/47] changes to default build config

We use the Android config as default setup

- Enable roaming and bgscan and nl80211
- Remove WPS NFC support
- Add required Mesh flags

Signed-off-by: Yaniv Machani <yanivma@ti.com>
---
 hostapd/android.config        |  2 +-
 wpa_supplicant/android.config | 16 +++++++++++++---
 2 files changed, 14 insertions(+), 4 deletions(-)

diff --git a/hostapd/android.config b/hostapd/android.config
index e14423f..6ee3a07 100644
--- a/hostapd/android.config
+++ b/hostapd/android.config
@@ -16,7 +16,7 @@
 #CONFIG_DRIVER_WIRED=y
 
 # Driver interface for drivers using the nl80211 kernel interface
-#CONFIG_DRIVER_NL80211=y
+CONFIG_DRIVER_NL80211=y
 # driver_nl80211.c requires a rather new libnl (version 1.1) which may not be
 # shipped with your distribution yet. If that is the case, you need to build
 # newer libnl version and point the hostapd build to use it.
diff --git a/wpa_supplicant/android.config b/wpa_supplicant/android.config
index 6536c11..237ba75 100644
--- a/wpa_supplicant/android.config
+++ b/wpa_supplicant/android.config
@@ -29,7 +29,7 @@
 #CONFIG_DRIVER_WEXT=y
 
 # Driver interface for Linux drivers using the nl80211 kernel interface
-#CONFIG_DRIVER_NL80211=y
+CONFIG_DRIVER_NL80211=y
 CONFIG_LIBNL20=y
 
 # QCA vendor extensions to nl80211
@@ -143,7 +143,7 @@ CONFIG_WPS_ER=y
 # registrar.
 #CONFIG_WPS_REG_DISABLE_OPEN=y
 # Enable WPS support with NFC config method
-CONFIG_WPS_NFC=y
+#CONFIG_WPS_NFC=y
 
 # EAP-IKEv2
 #CONFIG_EAP_IKEV2=y
@@ -466,7 +466,7 @@ CONFIG_HS20=y
 #CONFIG_MATCH_IFACE=y
 
 # Disable roaming in wpa_supplicant
-CONFIG_NO_ROAMING=y
+#CONFIG_NO_ROAMING=y
 
 # AP mode operations with wpa_supplicant
 # This can be used for controlling AP mode operations with wpa_supplicant. It
@@ -543,3 +543,13 @@ CONFIG_WIFI_DISPLAY=y
 #CONFIG_OWE=y
 
 include $(wildcard $(LOCAL_PATH)/android_config_*.inc)
+
+# bgscan
+NEED_BGSCAN=y
+CONFIG_BGSCAN_LEARN=y
+CONFIG_BGSCAN_SIMPLE=y
+
+
+
+CONFIG_MESH=y
+CONFIG_SAE=y
-- 
1.9.1

